templates/memoryusagegraph.mustache

```diff
@@ -182,14 +182,14 @@ function processGraph(id, data) {
               .style("opacity", .9);
 
           div.html(`
-              Sample #${d.sampleindex} <br/>
+              Sample ${d.sampleindex.toLocaleString('{{lang}}')} <br/>
               Memory: ${display_size(d.value)}
           `)
               .style("left", (event.pageX) + "px")
               .style("top", (event.pageY - 28) + "px");
 
           memoryusagedetails.innerHTML = `
-              Sample #${d.sampleindex}, Memory Usage: ${display_size(d.value)}
+              Sample ${d.sampleindex.toLocaleString('{{lang}}')}, Memory Usage: ${display_size(d.value)}
```

Comment:
As it's starting to do more, could we move this into a local `const` and share its usage?

Reply:
done